docs/specs/localization.yml

```diff
@@ -788,3 +787,75 @@ outputs:
     {"toc_rel": "toc.json"}
   docs/a/toc.json: |
     {"items":[{"toc_title": "yml reference a"}]}
+---
+# fallback to deleted token/codesnippet
+commands:
+  - build --locale zh-cn
+repos:
+  https://github.com/deletedtoken/a:
+    - files:
+        docfx.yml:
+      author: a
+    - files:
+        docfx.yml:
+        docs/tokena.md: |
+          [!include[include deleted tokenb](tokenb.md)
+          [!code-csharp[Main](program.cs)]
+        docs/tokenb.md: b
+        docs/program.cs:
+      author: b
+  https://github.com/deletedtoken/a.zh-cn:
+    - files:
+        docs/a.md: |
+          [!include[include deleted tokena](tokena.md)
+outputs:
+  docs/a.json: |
+    {"content":"<p>b</p>\n<pre><code class=\"lang-csharp\" name=\"Main\"></code></pre>"}
+---
+# fallback to deleted toc
+commands:
+  - build --locale zh-cn
+repos:
+  https://github.com/deletedtoc/a:
+    - files:
+        docfx.yml:
+      author: a
+    - files:
+        docfx.yml:
+        docs/a/TOC.yml: |
+          - href: a.md
+            name: reference markdown a
+      author: b
+  https://github.com/deletedtoc/a.zh-cn:
+    - files:
+        docs/b/TOC.yml: |
+          - href: ../a/TOC.yml
+            name: reference toc a
+        docs/a/a.md:
+outputs:
+  docs/a/a.json:
+  docs/b/toc.json: |
+    {"items":[{"toc_title":"reference toc a","children":[{"toc_title":"reference markdown a","href":"../a/a"}]}]}
+---
+# fallback to deleted inclusion(toc/codesnippet/token) only works for localization docset build
+commands:
+  - build
+repos:
+  https://github.com/onlyloc/a:
+    - files:
+        docfx.yml:
+        docs/b/TOC.yml: |
+          - href: ../a/TOC.yml
+            name: reference toc a
+        docs/a/a.md:
+      author: a
```

Comment:
author should be optional

Reply:
include _do not fallback token/codesnippet for english_ test

Reply:
ok